src/Sulu/Bundle/AdminBundle/Resources/js/components/Tabs/Tabs.js

```diff
@@ -228,14 +227,16 @@ class Tabs extends React.Component<Props> {
                 return this.childIndices;
             }
 
-            if (visibleWidth + nextWidth > this.tabsWrapperWidth) {
+            if (visibleWidth + nextWidth > this.tabsContainerWrapperWidth) {
                 break;
             }
 
             visibleWidth += nextWidth;
             visibleTabIndices = [...visibleTabIndices, index];
         }
 
+        // Since visibleTabIndices contains only the indices of the elements that can be fully shown, we need to reset
```

Comment:
Maybe also mention that the `selectedIndex` is still at the beginning, where it does not belong